package/Dockerfile

```diff
@@ -104,8 +104,8 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.6.0
-ENV CATTLE_DASHBOARD_UI_VERSION v2.6.0
+ENV CATTLE_UI_VERSION 2.6.0-rc1
```

Comment:
2.6.1-rc1, not 2.6.0?

Reply:
I had amended the commit and did not push yet :) my bad